packages/amplify-provider-awscloudformation/resources/rootStackTemplate.json

```diff
@@ -21,7 +21,22 @@
       "Type": "AWS::S3::Bucket",
       "DeletionPolicy": "Retain",
       "Properties": {
-        "BucketName": { "Ref": "DeploymentBucketName" }
+        "BucketName": {
+          "Ref": "DeploymentBucketName"
+        },
+        "VersioningConfiguration": {
```

Comment:
What is this property used for?

Reply:
We should test if there are any impact to existing projects by turning on  versioning for bucket